packages/api/src/utils/authenticateUser.ts

```diff
@@ -24,11 +24,11 @@ export const authenticateUser = async ({ data, req, res }: AuthenticateArgs) =>
   try {
     const existingUser = await em.findOne(User, { email: data.email });
     if (existingUser) {
-      req.session.id = existingUser.id;
+      req.session = { id: existingUser.id };
```

Comment:
Is there a reason this is better? Or just best practice?

Reply:
The change in types I made makes session either a `Session` or `null`; as a result, I would have had to do `req.session?.id =` and you can't assign that way: 

<img width="754" alt="image" src="https://github.com/AmericanAirlines/Hangar/assets/6445731/fb46d089-23ee-46ad-882c-938e67a98715">

The change in type was necessary because our cookie library doesn't like the removal of that id field and instead prefers that the entire session be set to `null` in order to clear it